lib/src/components/NxTextInput/NxTextInput.scss

```diff
@@ -116,6 +116,11 @@
       }
     }
 
+    .nx-text-input__input {
+      &::placeholder {
```

Comment:
before the placeholder text wasn't specifically set for disabled, so now it follows the mocks (grey-60 when disabled, grey-40 when not)